### ISSUE TITLE
Jackett plugin gives connection error, even though properly configured

### ISSUE THREAD

**Issue by COOLak** [qbittorrent/search-plugins#273]
![Screenshot_1](https://github.com/qbittorrent/search-plugins/assets/8677627/b666f0c9-02a4-4d40-a5f8-bfe2298be4b8)
![Screenshot_2](https://github.com/qbittorrent/search-plugins/assets/8677627/46f43e4d-69d5-4399-9023-b3bf02e3fa81)
![Screenshot_3](https://github.com/qbittorrent/search-plugins/assets/8677627/75dc2f32-10a7-4f9e-a43a-73843e424505)

As can be seen from the screenshots, everything is configured properly. I've added my API key to the file %localappdata%\qBittorrent\nova3\engines\jackett.json.

The IP and port of Jackett are also correct. Still "connection error". In web interface of Jacket itself, there are no isssues, and the search works.

Please advise?

**Comment by sourcelocation** [comment]
Same issue, have you found a solution?

**Comment by Piccirello** [comment]
Are you using a proxy? If so, you may need to disable "Use proxy for general purposes" in qBittorrent options.

**Comment by COOLak** [comment]
> Are you using a proxy? If so, you may need to disable "Use proxy for general purposes" in qBittorrent options.

No, I'm not using a proxy. And I still haven't found a solution.

**Comment by makoto-kokoroki** [comment]
also having this problem since a few months ago;

**Comment by jiznon** [comment]
Also having this issue, but looking through the logs shows:

> <html><body>

  | Jackett.Common.IndexerException: Exception (torlock): Request to https://www.torlock2.com/all/torrents/finding-pictures-documentary.html?sort=added&order=desc failed (Error 522) - The tracker seems to be down.
-- | --



</body>
</html>

Trying to visit that tracker's site does indeed show it's down. My advice is to confirm that these sites are indeed down, and remove them if they aren't expected back up.

**Comment by makoto-kokoroki** [comment]
I figured it out.
I am a dumb ass.
Every time I was trying to figure it out? One little character eluded me... The letter 's.'
I kept the fuckin link all gonked up each time with 'https' as I had it set up this way for awhile, and I just kept overlooking it each time. 
My apologies.
-------- Original Message --------On 5/15/24 6:33 PM, jiznon  wrote:
Also having this issue, but looking through the logs shows:







 
Jackett.Common.IndexerException: Exception (torlock): Request to https://www.torlock2.com/all/torrents/finding-pictures-documentary.html?sort=added&order=desc failed (Error 522) - The tracker seems to be down.






Trying to visit that tracker's site does indeed show it's down. My advice is to confirm that these sites are indeed down, and remove them if they aren't expected back up.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/qbittorrent/search-plugins/issues/273#issuecomment-2113591947",
"url": "https://github.com/qbittorrent/search-plugins/issues/273#issuecomment-2113591947",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by jiznon** [comment]
Hey, we got two solutions to very similar problems.
